COPYING

```diff
@@ -0,0 +1,379 @@
+This software is available to you under a choice of one of two
+licenses.  You may choose to be licensed under the terms of the the
+BSD license or the GNU General Public License (GPL) Version
+2, both included below.
+
+Copyright (c) Intel Corporation.  All rights reserved.
```

Comment:
Does this need a year?

Reply:
should not, this is a copy of libfabric as they use the same license.